importer/specobject/src/test/java/org/itsallcode/openfasttrace/importer/specobject/TestSpecobjectImporter.java

```diff
@@ -299,4 +309,105 @@ void testImportProvidesCoverage()
         verify(listenerMock).endSpecificationItem();
         verifyNoMoreInteractions(listenerMock);
     }
+
+    @Test
+    void testCustomTagsWithoutNamespacesLogsWarning()
+    {
+        final RecordingLogHandler logHandler = RecordingLogHandler.registerRecordingLogHandler();
```

Comment:
Always de-register the log handler after the test, e.g. with try-finally or create a Junit5 extensions -- or check if an extension already exists.

This would be a good fit for https://github.com/itsallcode/junit5-system-extensions

Reply:
Sophisticate but I also found an elegant solution that prevents all other tests from cleaning up the root logger.